torch/_inductor/lowering.py

```diff
@@ -2745,7 +2747,7 @@ def is_aligned(x):
 make_fallback(aten._grouped_mm, require_dense)
 
 # Need templated kernel. Probably impossible to write efficiently
-make_fallback(aten.convolution_backward, constrain_to_fx_strides)
+make_fallback(aten.convolution_backward, constrain_to_fake_tensors)
```

Comment:
Pretty sure for backward we want to keep the same dimension dense, but otherwise, we do not need to match exactly (current behavior)

Reply:
Gotcha! I was experimenting with what would happen if I just switched all of them, but clearly that didn't work. :-) I'll do a second pass and see if I can more precisely audit which needs to be changed.